src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,64 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/todo';
+import usersFromServer from '../../api/users';
```

Comment:
Avoid directly importing `usersFromServer` in this component. Instead, pass the necessary user data as props from the parent component (`App` component). This will ensure that data preparation is centralized and aligns with the task requirements.